solution.cpp

```diff
@@ -1,18 +1,128 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
+ * Copyright &copy; 2015 (Dong Kim) <donm93@csu.fullerton.edu>
  * Released under the [MIT License] (http://opensource.org/licenses/MIT)
  * ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * 
+ * this program is to convert the integer numbers to roman numerals and then ask to users to do it again or not.
  */
 
-// TODO: `#include`s and function prototypes
+//    
+             #include <iostream>
+             #include <string>
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+int main () {
+
+	std::string roman;
+	int integer;
+	int number;
+	char repeat;
+
+	do{
+		std::cout << "Please Enter an Integer: ";
+		std::cin >> integer;
+		if (integer >= 1000) 
+		{
+			number = (integer /1000);
+			for (int i=0; i< number; i++)
+			{
+				roman += 'M';
+			}
+		integer %= 1000;
+		}
+		else if (integer >=100)
```

Comment:
`else` probably not necessary here.  in fact, i suspect that it caused you to have to duplicate a lot of code.